ToDo.Core/Services/INativeViewService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xamarin.Forms;
+
+namespace ToDo.Core.Services
+{
+    public interface INativeViewService
+    {
+        int[] GetCoordinates(VisualElement visualElement);
```

Comment:
I'll change that array, I like the anonymous type idea. Or I could return just an int because I only need the Y position. I decided to use the array (I mean return not just the Y) because it seemed more reusable to me.

Reply:
Yes, good thinking :+1: But a type is always better than an array, because it is more descriptive. You know you get an x and y property, and not a random `IndexOutOfBoundsException` ;)

Reply:
I decided to return just an integer.

52ede3fd3185ab1dbed1941b8b2e0c956604f146